policy_definitions/network_security/policy_set_definition_qby_NetworkSecurity.json

```diff
@@ -0,0 +1,22 @@
+{
+    "name": "QBY-NetworkSecurity",
+    "type": "Microsoft.Authorization/policySetDefinitions",
+    "apiVersion": "2021-06-01",
+    "scope": null,
+    "properties": {
+        "policyType": "Custom",
+        "description": "Enforce namingconform NSG to have a rule that denys all inbound traffic in Vnet. Ensures all subnets and NSG to be namingconform and with their Vnet in name associated",
```

Comment:
The goal of this set is to enforce, that any intra vNET traffic is denied. The naming and deny rule setc. are just means to an end.